dappnode/DNP_DAPPMANAGER
# Move and rename getEndpointConfig()

## Context

getEndpointConfig() method was missplaced in the code and had a non intuitive name
## Approach

renamed and moved the method to "packages/dappmanager/src/modules/docker/api/network.ts" a file related to docker API calls to docker network
## Test instructions

on _/packages/dappmanage_ dir, run `yarn run test:int` to execute unit test related to this new renamed method `getDnCoreNetworkContainerAliases(containerName)`. This will verify that when calling this method with the dappmanager container, it returns a network (dn_core) confuguration and it is not null
-->
